test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -1331,7 +1331,7 @@ try {
             ($observed.OsFreeVirtualMemory -gt 0) | Should Be $true
         }
 
-        It "(special case) Test for property = OsLocalDateTime" {
+        It "(special case) Test for property = OsLocalDateTime" -Pending:$true {
```

Comment:
OsLocalDateTime return empty to me. And the test is wrong. Yes, we should open a new Issue...